integration-tests/messages/addIPFSMessage.test.ts

```diff
@@ -1,52 +1,73 @@
 import "@frequency-chain/api-augment";
 import { ApiRx } from "@polkadot/api";
+import type { Codec } from '@polkadot/types-codec/types';
 import { connect, createKeys } from "../scaffolding/apiConnection"
 import { KeyringPair } from "@polkadot/keyring/types";
 import { createSchema, addIPFSMessage, createMsa } from "../scaffolding/extrinsicHelpers";
 import { PARQUET_BROADCAST } from "../schemas/fixtures/parquetBroadcastSchemaType";
 import assert from "assert";
-import { GenericEvent } from "@polkadot/types";
-import { EventMap } from "../scaffolding/helpers";
-import { Event } from "@polkadot/types/interfaces";
+import { AccountFundingInputs, createAndFundAccount, DevAccounts, EventMap, generateFundingInputs, txAccountingHook } from "../scaffolding/helpers";
+
+describe("Add Offchain Message", function () {
+    this.timeout(15000);
+
+    let fundingInputs: AccountFundingInputs;
 
-describe("Add Offchain Message", () => {
     let api: ApiRx;
     let keys: KeyringPair;
+    let schemaId: Codec;
+
+    // This is how the IPFS data was created:
+    //
+    // echo "This is a test of Frequency." > frequency_test
+    // % ipfs add frequency_test
+    // added QmYzm8KGxRHr7nGn5g5Z9Zv9r8nN5WNn7Ajya6x7RxmAB1 frequency_test
+    // % wc -c frequency_test
+    // 29 frequency_test
+    const ipfs_cid = "QmYzm8KGxRHr7nGn5g5Z9Zv9r8nN5WNn7Ajya6x7RxmAB1";
```

Comment:
@JoeCap08055 If you work on #786 it will fail this test. Not an issue for this test (as it is correct for the test), but heads up for that issue.

Reply:
Cool. Will make a good test case.